bin/rubocop-diff

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Why bash, do you use any features which are in bash but not in the bourne shell?  The latter was not affected by [shellshock](https://en.wikipedia.org/wiki/Shellshock_(software_bug))